README.md

```diff
@@ -1,3 +1,21 @@
 # hackmud_wiki
 
 [![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC_BY--NC--SA_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)
+
+## Setting up a local environment
+
+### Prerequisites
+
+- [pnpm](https://pnpm.io/installation)
+
+### Installation
+
+1. Clone the repository
+2. Run `pnpm install` in the root directory
+3. Set your Node.js runtime to version >= `20.x` (as specified in `.package.json`)
```

Comment:
And, bringing you along on my journey-- the above error can't be fixed by `pnpm env` because pnpm can't run. (chicken and egg)

I needed to use nvm to install a compatible version, run `pnpm env` then I need to uninstall all versions from nvm?

This is a very significant dev machine workflow change-- esp. if people are doing work for other projects which have different dev environments. Eventually, we'll do all (or most) of our dev work in codespaces, but we can't afford that yet. I'm happy with making the switch on my dev machine to try things out, but I don't think it's a good idea force the whole team to move over right now. (additionally because pnpm doesn't support project local node version switching with `pnpm env` yet.)

Reply:
Would you suggest we add a note about switching to the correct node version using your version manager of choice? With instructions for `nvm`

It seems like `nvm`, `n` and `pnpm` all seem to wrestle for control of managing your node version.

I experienced a similar issue to you where `pnpm env` caused me unable to switch my node version using `n`

Reply:
for now, let's just leave the instructions vague re: node version management.
We can update the end of the line to have this though:
`(as specified in "engines" in [./package.json](./package.json))`

Reply:
Updated